src/index/sparse/sparse_inverted_index_config.h

```diff
@@ -43,22 +43,31 @@ class SparseInvertedIndexConfig : public BaseConfig {
             .for_search()
             .for_range_search();
         /**
-         * The term frequency part of score of BM25 is:
-         * tf * (k1 + 1) / (tf + k1 * (1 - b + b * (doc_len / avgdl)))
-         * as more documents being added to the collection, avgdl can also
-         * change. In WAND index we precompute and cache this score in order to
-         * speed up the search process, but if avgdl changes, we need to
-         * re-compute such score which is expensive. To avoid this, we upscale
-         * the max score by a ratio to compensate for avgdl changes. This will
-         * make the max score larger than the actual max score, it makes the
-         * filtering less aggressive, but guarantees the correctness.
-         * The larger the ratio, the less aggressive the filtering is.
+         * wand_bm25_max_score_ratio is assigned two functions:
```

Comment:
The term frequency part of score of BM25 is:... . but if avgdl changes, the max score changes. this can be used to adjust the max scored used to compute the max score.
1. if set to a value greater than 1, ...
2. if set to a value less than 1, ...